deployment/uberspace-ansible/deploy.yml

```diff
@@ -0,0 +1,114 @@
+---
+- hosts: servers
+  vars:
+    install_root: ~/
+    project_repo: https://github.com/c3foc/hagrid.git
+  gather_facts: false
+
+  tasks:
```

Comment:
It is highly recommended to create structured roles opposed to simply putting a whole lot of tasks into the host files. This is due to
 * better structuring of the playbook
 * easier maintenance of the cluster
 * integrity of the setup across multiple hosts

Putting tasks into a host.yml file is only meant for tasks that only need to be run for that exact host and aren't reoccurring.

But as you want to separate the website code and the provisioning code into different repos anyway (you don't want to serve the provisioning code aren't you?) this might work for now.

Reply:
I'm creating a seperate repository. Lets continue the discussion there :)